src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.ReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReaderServiceImplTest {
+
+    private static ReaderService readerService;
+    private static List<String> expected;
+
+    @BeforeAll
+    public static void init() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test
+    void reader_fileNotExist_notOk() {
+        expected = new ArrayList<>();
+        String nonExistingFile = "not_exist";
+        assertThrows(RuntimeException.class,
+                () -> readerService.readFromFile(nonExistingFile),
+                "Should throw runtime exception.");
+    }
+
+    @Test
+    void reader_fileIsEmpty_notOk() {
+        expected = new ArrayList<>();
+        expected = readerService.readFromFile("src/main/resources/input_file.csv");
+        assertThrows(AssertionError.class,
+                () -> assertTrue(expected.isEmpty()),
+                "Should throw runtime exception.");
+    }
+
+    @Test
+    void reader_readFromFile_ok() {
+        expected = readerService.readFromFile("src/main/resources/input_file.csv");
```

Comment:
```suggestion
        List<String> actual = readerService.readFromFile("src/main/resources/input_file.csv");
```
fix the same issue in all places. And also from common mistakes:
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/53354844/7c52f5e2-77a8-4f53-8011-9b3967ceffe1)